packages/venia-ui/lib/components/Checkout/addressForm.js

```diff
@@ -130,7 +125,7 @@ const AddressForm = props => {
                 </div>
             </div>
             <div className={classes.footer}>
-                <Button onClick={cancel}>Cancel</Button>
+                <Button onClick={onCancel}>Cancel</Button>
```

Comment:
@jimbo see how I'm passing the function directly here? Based on our other conversation I should change these to be wrapped/returned by the talon.